src/loaders/lottie/tvgLottieModel.cpp

```diff
@@ -31,7 +31,16 @@
 /* Internal Class Implementation                                        */
 /************************************************************************/
 
+static inline float _applySmoothness(float value, float smoothness)
```

Comment:
If the logic of this function is not very lengthy and it won’t be called from multiple places, there’s no need to separate it into its own function. I think you can merge it in `factor()` with comment `//apply smoothness`

Reply:
done